src/electron/views/screenshot/screenshot-view.tsx

```diff
@@ -1,13 +1,31 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { NamedFC } from 'common/react/named-fc';
+import { ScreenshotData } from 'common/types/store-data/unified-data-interface';
+import { isEmpty } from 'lodash';
 import * as React from 'react';
 
-export type ScreenshotViewProps = {};
-export const ScreenshotView = NamedFC<ScreenshotViewProps>('ScreenshotView', props => {
-    return (
-        <div>
-            <h1 tabIndex={0}>This is the screenshot view</h1>
-        </div>
-    );
+import { Screenshot } from './screenshot';
+
+export type ScreenshotViewProps = {
+    screenshotData: ScreenshotData;
+};
+
+const screenshotAltText: string = 'axe android results screenshot';
```

Comment:
Maybe out of scope for this PR, but we should think about where to get the alt text for the screenshot and if this default alt text is sufficient.

Reply:
I agree; where should we get the alt-text from @ferBonnin and @dbjorge? Or is hard coding it ok for now?

Reply:
mmm I might add something like 
"Automated checks results highlighted in a screenshot of the app"

Reply:
Updated in the most recent commit

Reply:
If the alt text is not being use anywhere else, it should be fine here. We don't have a proper "resources" system right now.